.github/workflows/ci_action.yml

```diff
@@ -11,31 +11,30 @@ env:
 
 jobs:
 
-  check-code-black:
+  check-code-black-isort-flake8:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout branch
         uses: actions/checkout@v2
         with:
           ref: ${{ env.CHECKOUT_BRANCH }}
 
-      - name: Check code compliance with black
-        run: |
-          pip install black[jupyter]
-          black . --check --diff
+      - name: Prepare linters
+        run: pip install black[jupyter] isort flake8 nbqa
 
-  check-code-isort:
-    runs-on: ubuntu-latest
-    steps:
-      - name: Checkout branch
-        uses: actions/checkout@v2
-        with:
-          ref: ${{ env.CHECKOUT_BRANCH }}
+      - name: Check code compliance with black
+        run: black . --check --diff
 
       - name: Check code compliance with isort
         run: |
-          pip install isort
           isort . --check --diff
+          nbqa isort examples --check --diff
+
+      - name: Check code compliance with flake8
+        run: |
+          pip install isort
+          flake8 .
+          nbqa flake8 examples --ignore=E402
```

Comment:
- `pip install isort` should be removed,
- I would also here change `examples` to `.` so that we have `nbqa flake8 . --ignore=E402`